src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -1788,6 +1802,37 @@ public void RegisterGeolocationPermissionCallback(GeolocationPermissionCallback
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Creates a new hit test.
+        /// <param name="x">the horizontal position to query</param>
+        /// <param name="y">the vertical position to query</param>
+        /// <param name="mode">the mode of hit test</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebHitTest CreateHitTest(int x, int y, WebHitTest.HitTestMode mode)
+        {
+            System.IntPtr result = Interop.WebView.CreateHitTest(SwigCPtr, x, y, (int)mode);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return new WebHitTest(result, true);
+        }
+
+        /// <summary>
+        /// Creates a hit test asynchronously.
+        /// <param name="x">the horizontal position to query</param>
+        /// <param name="y">the vertical position to query</param>
+        /// <param name="mode">the mode of hit test</param>
+        /// <param name="callback">the callback function</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool CreateHitTestAsynchronously(int x, int y, WebHitTest.HitTestMode mode, HitTestCreatedCallback callback)
```

Comment:
Ditto. `HitTestAsynchronously`.

Reply:
Done.

Reply:
I think the term "HitTestCreatedCallback" also be just "HitTestCallback" or "HitTestDoneCallback" or "HitTestResultCreatedCallback".
And please add more description on callback param.
```C#
/// <param name="callback">the callback function to be called when the hit test is done.</param>
```

Reply:
I changed HitTestCreatedCallback to HitTestFinishedCallback.
If not ok, I will change it again according to your comment.

Reply:
Great. HitTestFinished is best. Thank you.